fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,668 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "FCSA Service Request",
+    "description": "FCSA Service Request",
+    "contact": {
+      "email": "dee@opencitieslab.org"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "2.1"
+  },
+  "servers": [
+    {
+      "url": "https://qaeservices1.capetown.gov.za/coct/api/zcur-guest/login",
+      "description": "FCSA Service Request API"
```

Comment:
I think it would be more appropriate to describe this as the QA server

Reply:
I have changed the description to FCSA Service Request API

Reply:
sure, we can discuss this with the relevant people from IS&T when we interact with them